app/queries/low_inventory_query.rb

```diff
@@ -7,13 +7,13 @@ def self.call(organization)
     items.each do |item|
       quantity = inventory.quantity_for(item_id: item.id)
       if quantity < item.on_hand_minimum_quantity.to_i || quantity < item.on_hand_recommended_quantity.to_i
-        low_inventory_items.push(OpenStruct.new(
+        low_inventory_items.push({
```

Comment:
If we're already not using an OpenStruct, I'd much prefer defining a Data class and using dot notation. It's a lot safer than hash access. Hashes are really supposed to be for dynamic/unknown values, not specified fields.

Reply:
Fixed in [bb94402](https://github.com/rubyforgood/human-essentials/pull/4832/commits/bb944022021f266f1a95564c3564f2765f74eb30).